hack/debug-network.sh

```diff
@@ -0,0 +1,574 @@
+#!/bin/bash
+
+# echoes args to stderr and exits
+die () {
+    echo "$*" 1>&2
```

Comment:
We have `os::log::error` -- I'm not sure we *really* need this as a helper function